openshift/sdk-pv.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
-1 for having this. We should recommend users to use oc-cluster wrapper:
https://github.com/openshift-evangelists/oc-cluster-wrapper